Mods/CombatExtended/Languages/Russian/DefInjected/RulePackDef/RulePacks_Swearing.xml

```diff
@@ -1,10 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <LanguageData>
+	<!--А это зачем переводить?...-->
+	<!--	<AttackMote.rulePack.rulesStrings>
+		<li>rule->[swear]</li>
+	</AttackMote.rulePack.rulesStrings>
+	
+	<SuppressedMote.rulePack.rulesStrings>
+		<li>rule->[suppressed]</li>
+	</SuppressedMote.rulePack.rulesStrings>
+-->
 
-  <!-- EN: rule->[swear] -->
-  <AttackMote.rulePack.rulesStrings.0>rule->[swear]</AttackMote.rulePack.rulesStrings.0>
-  <!-- EN: rule->[suppressed] -->
-  <SuppressedMote.rulePack.rulesStrings.0>rule->[suppressed]</SuppressedMote.rulePack.rulesStrings.0>
```

Comment:
confirm removal?

Reply:
top comment says "why translate this?"
also the rule string only has tagged string `[swear]` & `[suppressed]`, that should be translated somewhere else